Modules/FIT/FIT/src/MIPCheck.cxx

```diff
@@ -0,0 +1,382 @@
+  }
+  mNPeaksToFit = std::stoi(mCustomParameters.atOrDefaultValue("nPeaksToFit", "2"));
+  mFitRangeLow = std::stof(mCustomParameters.atOrDefaultValue("fitRangeLow", "11.0"));
+  mFitRangeHigh = std::stof(mCustomParameters.atOrDefaultValue("fitRangeHigh", "35.0"));
+
+  std::string parseableParam;
+
+  parseableParam = mCustomParameters.atOrDefaultValue("gausParamsMean", "");
+  mGausParamsMeans = helper::parseParameters<float>(parseableParam, ",");
+  if (mGausParamsMeans.size() != mNPeaksToFit) {
+    ILOG(Warning, Support) << "Initial fit arguments for gaussian means not provided for all MIP peaks. Trying to estimate them..." << ENDM;
+  }
+
+  parseableParam = mCustomParameters.atOrDefaultValue("gausParamsSigma", "3.0, 7.0");
+  mGausParamsSigmas = helper::parseParameters<float>(parseableParam, ",");
+
+  parseableParam = mCustomParameters.atOrDefaultValue("meanWarningsLow", "");
+  mMeanWarningsLow = helper::parseParameters<float>(parseableParam, ",");
+
+  parseableParam = mCustomParameters.atOrDefaultValue("meanWarningsHigh", "");
+  mMeanWarningsHigh = helper::parseParameters<float>(parseableParam, ",");
+
+  parseableParam = mCustomParameters.atOrDefaultValue("meanErrorsLow", "");
+  mMeanErrorsLow = helper::parseParameters<float>(parseableParam, ",");
+
+  parseableParam = mCustomParameters.atOrDefaultValue("meanErrorsHigh", "");
+  mMeanErrorsHigh = helper::parseParameters<float>(parseableParam, ",");
+
+  parseableParam = mCustomParameters.atOrDefaultValue("sigmaWarnings", "");
+  mSigmaWarnings = helper::parseParameters<float>(parseableParam, ",");
+
+  parseableParam = mCustomParameters.atOrDefaultValue("sigmaErrors", "");
+  mSigmaErrors = helper::parseParameters<float>(parseableParam, ",");
+
+  parseableParam = mCustomParameters.atOrDefaultValue("drawMeanWarningsLow", "false, false");
+  mDrawMeanWarningsLow = helper::parseParameters<bool>(parseableParam, ",");
+
+  parseableParam = mCustomParameters.atOrDefaultValue("drawMeanWarningsHigh", "false, false");
+  mDrawMeanWarningsHigh = helper::parseParameters<bool>(parseableParam, ",");
+
+  parseableParam = mCustomParameters.atOrDefaultValue("drawMeanErrorsLow", "true, true");
+  mDrawMeanErrorsLow = helper::parseParameters<bool>(parseableParam, ",");
+
+  parseableParam = mCustomParameters.atOrDefaultValue("drawMeanErrorsHigh", "true, true");
+  mDrawMeanErrorsHigh = helper::parseParameters<bool>(parseableParam, ",");
+
+  parseableParam = mCustomParameters.atOrDefaultValue("drawSigmaWarnings", "false, false");
+  mDrawSigmaWarnings = helper::parseParameters<bool>(parseableParam, ",");
+
+  parseableParam = mCustomParameters.atOrDefaultValue("drawSigmaErrors", "false, false");
+  mDrawSigmaErrors = helper::parseParameters<bool>(parseableParam, ",");
+
+  parseableParam = mCustomParameters.atOrDefaultValue("labelPos", "0.15, 0.2, 0.85, 0.45");
+  mVecLabelPos = helper::parseParameters<double>(parseableParam, ",");
```

Comment:
btw, you can patch parseParameters method and use `mCustomParameters.atOrDefaultValue` inside. Less code lines